tools/wpt/expectation.json

```diff
@@ -2257,6 +2257,7 @@
     },
     "idlharness-shadowrealm.window.html": false,
     "idlharness.any.worker.html": [
+      "Event interface object length",
```

Comment:
Because of the default argument removal

Reply:
Yeah, you're right, missed that, thought it failed because of isTarget